src/components/core/core.js

```diff
@@ -264,70 +257,50 @@ export default class Core extends UIObject {
     return container
   }
 
-  setupMediaControl(container) {
-    if (this.mediaControl) { this.mediaControl.setContainer(container) } else {
-      this.mediaControl = this.createMediaControl($.extend({ container: container, focusElement: this.el }, this.options))
+  setupMediaControl() {
+    if (this.mediaControl) {
       this.listenTo(this.mediaControl, Events.MEDIACONTROL_FULLSCREEN, this.toggleFullscreen)
       this.listenTo(this.mediaControl, Events.MEDIACONTROL_SHOW, this.onMediaControlShow.bind(this, true))
       this.listenTo(this.mediaControl, Events.MEDIACONTROL_HIDE, this.onMediaControlShow.bind(this, false))
     }
   }
 
-  createMediaControl(options) {
-    if(options.mediacontrol && options.mediacontrol.external)
-      return new options.mediacontrol.external(options).render()
-    else
-      return new MediaControl(options).render()
-
-  }
-
   getCurrentContainer() {
-    if (!this.mediaControl || !this.mediaControl.container)
-      return this.containers[0]
-
-    return this.mediaControl.container
+    return this.activeContainer
```

Comment:
I think we could mark this function as "deprecated",  to discourage using it in favor of using `activeContainer`.